### PR TITLE
Upgrades + move to github package registry

### PR DESCRIPTION
**Pull request by Fjandin**
- Move to github package registry
- Upgrades

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -5,5 +5,6 @@ module.exports = {
     singleQuote: true,
     bracketSpacing: false,
     jsxBracketSameLine: false,
-    arrowParens: 'always'
+    arrowParens: 'always',
+    trailingComma: 'none'
```
Comment (erwinw) [inline .prettierrc.js]: 👎🏻

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.4.0]
+### Changed
+ - Move to github package registry
+ - Upgrade peer dependency `aws-sdk`
```
Comment (erwinw) [inline CHANGELOG.md:6]: Great! When are you moving to the v3 SDK?

File: jest.config.js [hunk 3]
```diff
@@ -2,7 +2,6 @@ module.exports = {
     rootDir: __dirname,
     verbose: false,
     silent: false,
-    browser: false,
```
Comment (erwinw) [inline jest.config.js:5]: You're using this in a browser now?

File: package.json [hunk 4]
```diff
@@ -29,22 +29,22 @@
...
-    "ts-jest": "^25.2.0",
-    "typescript": "^3.7.5"
+    "jest": "^26.6.3",
+    "prettier": "^2.2.1",
+    "ts-jest": "^26.5.5",
+    "typescript": "^4.2.4"
   },
   "peerDependencies": {
-    "aws-sdk": "^2.644.0"
+    "aws-sdk": "2.617.0"
```
Comment (erwinw) [inline package.json]: Why are you pinning at this exact version -- and an older version at that?

**Review by erwinw — APPROVED** [review]
LGTM

File: jest.config.js (hunk 3 above)
Comment (Fjandin) [reply jest.config.js:5]: nah.... this config value is just deprecated

**Review by erwinw — APPROVED** [review]
Luv it! Comma all the things,!

**Pull request merged**